File: Programmers/[12987] 숫자 게임/woong-jae/12987.js
function solution(A, B) {
  A.sort((a, b) => a - b);
  B.sort((a, b) => a - b);
  
  let answer = 0, indexA = 0;
  for(let i = 0; i < B.length; i++) {
      if(A[indexA] < B[i]) {
          answer += 1;
          indexA += 1;
      }
  }
  return answer;
}

File: Programmers/[12987] 숫자 게임/ONE/12987.java
import java.util.Arrays;

class Solution {
    public int solution(int[] A, int[] B) {
        int answer = 0, N = A.length;

        Arrays.sort(A);
        Arrays.sort(B);

        for (int indexA = N - 1, indexB = N - 1; indexA >= 0; indexA--)
            if (B[indexB] > A[indexA]){
                answer++;
                indexB--;
            }
        return answer;
    }
}